sdk/maps/maps-timezone-rest/test/public/mapsTimezone.spec.ts

```diff
@@ -0,0 +1,186 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Recorder, env } from "@azure-tools/test-recorder";
+import { isNodeLike } from "@azure/core-util";
+import { createTestCredential } from "@azure-tools/test-credential";
+import { assert } from "chai";
+import { createClient, createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import { MapsTimezone, isUnexpected } from "../../src";
+import { IanaIdOutput } from "../../src";
+
+describe("Authentication", function () {
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("should work with Microsoft Entra ID authentication", async function () {
+    
+    if (!isNodeLike) this.skip();
```

Comment:
Suggest to add comment like other packages why we are skipping the test in browser
```suggestion
    /**
     * Skip this test in browser because we have to use InteractiveBrowserCredential in the browser.
     * But it requires user's interaction, which is not testable in karma.
     * */
    if (!isNodeLike) this.skip();
```